### ISSUE TITLE
Node crashes if azure blob url is not reachable.

### ISSUE THREAD

**Issue by ashisherc** [mckalexee/multer-azure#3]
If blob storage is not reachable, exp. internet is not present while developing in local, The node app crashes. Please see the error details below.

```
[2017-09-27T12:39:44.667Z]  INFO: xxx-yyy-service/15991 on ashisherc: Listening on port 3001
[2017-09-27T12:39:54.889Z] ERROR: xxx-yyy-service/15991 on ashisherc: getaddrinfo ENOTFOUND xxx-yyy.blob.core.windows.net xxx-yyy.blob.core.windows.net:443 (err.code=ENOTFOUND)
    Error: getaddrinfo ENOTFOUND xxx-yyy.blob.core.windows.net xxx-yyy.blob.core.windows.net:443
        at errnoException (dns.js:28:10)
        at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:76:26)
/home/ashish/xxx-yyy-service/node_modules/multer-azure/index.js:15
                throw error;
                ^

Error: getaddrinfo ENOTFOUND xxx-yyy.blob.core.windows.net xxx-yyy.blob.core.windows.net:443
    at errnoException (dns.js:28:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:76:26)
[nodemon] app crashed - waiting for file changes before starting...
```

**Comment by ashisherc** [comment]
```
Blob.prototype.createContainer = function (name) {
        this.blobSvc.createContainerIfNotExists(name, function (error, result, response) {
            if (error) {
                throw error;
            }
        });
    };
```

The throw is not handled. I have created a PR to fix this issue. If this error happens it is handled in callback and will be cashed by nodejs error handler.

**Comment by mckalexee** [comment]
Pull Request #4 has been merged.

Thanks for contributing!

**Comment by mckalexee** [comment]
This is now available on NPM as [v2.0.1](https://www.npmjs.com/package/multer-azure).
